pages/common/pnmnorm.md

```diff
@@ -0,0 +1,21 @@
+# pnmnorm
+
+> Normalize the contrast in a PNM image.
+> See also: `pnmhisteq`.
+> More information: <https://netpbm.sourceforge.net/doc/pnmnorm.html>.
+
+- Force the brightest pixels to be white, the darkest pixels to be black and spread out the ones inbetween linearly:
+
+`pnmnorm {{path/to/image.pnm}} > {{path/to/output.pnm}}`
+
+- Force the brightest pixels to be white, the darkest pixels to be black and spread out the ones inbetween quadratically such that pixels with a brightness of `n` become 50 percent bright:
```

Comment:
```suggestion
- Force the brightest pixels to be white, the darkest pixels to be black and spread out the ones in between quadratically such that pixels with a brightness of `n` become 50 % bright:
```
Maybe we can go with the symbol here as the line is already long, and could probably cause certain translations to cross the 256-character limit.